src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Address.java

```diff
@@ -40,35 +46,62 @@ public Address(@JsonProperty("line1") String line1,
                    @JsonProperty("county") String county,
                    @JsonProperty("postcode") String postcode,
                    @JsonProperty("postcodeLookup") String postcodeLookup,
-                   @JsonProperty("postcodeAddress") String postcodeAddress) {
+                   @JsonProperty("postcodeAddress") String postcodeAddress,
+                   @JsonProperty("ukPortOfEntry") UkPortOfEntry portOfEntry,
+                   @JsonProperty("country") String country,
+                   @JsonProperty("inUk") YesNo isInUk) {
```

Comment:
Do we need an additional constructor for backwards compatibility or are we using builders everywhere?

Reply:
I built and imported it on tribunals api for checks and dev for international flag and had no errors from this class. Only Identity needed another constructor

Reply:
I added another constructor of the original just in case theres a service I havent looked at